Block/RecentPostsBlockService.php

```diff
@@ -34,6 +34,11 @@ class RecentPostsBlockService extends BaseBlockService
     protected $manager;
 
     /**
+     * @var Pool
+     */
+    private $adminPool;
```

Comment:
@core23 except this change I would merge this PR as it is, can you get in touch with it?